Tools/scripts/configure-ci.sh

```diff
@@ -83,5 +83,5 @@ fi
 
 . ~/.profile
 
-pip install --user -U argparse empy pyserial pexpect future lxml
+pip install --user -U argparse empy pyserial pexpect future lxml numpy
```

Comment:
running sitltest on semaphoreci (we currently run it on travis): https://semaphoreci.com/lucasdemarchi/ardupilot/branches/pr-px4-build/builds/3